src/components/footer.js

```diff
@@ -1,20 +1,44 @@
 /** @jsx jsx */
 import { jsx } from 'theme-ui'
-import { Container } from '@theme-ui/components'
+import { Flex, Container } from '@theme-ui/components'
 import { Link } from 'gatsby'
+import { Monogram } from '../components/logo'
 
 const Footer = () => (
   <footer
     sx={{
+      height: 80,
       borderTop: `1px solid`,
       borderTopColor: `border`,
       pb: [`5`, `0`],
     }}
   >
-    <Container sx={{ variant: `layout.container.footer` }}>
-      <span>
-        <Link to="/">Recipe Palette</Link> made with 🧡
-      </span>
+    <Container>
+      <Flex
+        sx={{
+          my: [`2`, `3`],
+          flexDirection: `row`,
+          alignItems: `center`,
+          justifyContent: `space-between`,
+        }}
+      >
+        <div>
+          <Link to="/">
+            <Monogram height={36} />
+          </Link>
+        </div>
+        <div sx={{ fontSize: `1`, color: `gray` }}>
+          Copyright 2020 &copy; Recipe Palette
+        </div>
+        <div>
+          <a
+            sx={{ variant: `buttons.secondary` }}
+            href="mailto:admin@recipepalette.com"
+          >
+            Contact Us
+          </a>
+        </div>
+      </Flex>
```

Comment:
This looks a little wonky on mobile. I think that there needs to be some kind of padding so the footer doesn't get hidden behind the navigation.